ts/features/bonus/cdc/navigation/CdcStackNavigator.tsx

```diff
@@ -0,0 +1,35 @@
+import { createStackNavigator } from "@react-navigation/stack";
+import * as React from "react";
+import CdcBonusRequestInformationTos from "../screens/CdcBonusRequestInformationTos";
+import CdcBonusRequestSelectYear from "../screens/CdcBonusRequestSelectYear";
+import CdcBonusRequestSelectResidence from "../screens/CdcBonusRequestSelectResidence";
+import CdcBonusRequestBonusRequested from "../screens/CdcBonusRequestBonusRequested";
+import { CdcBonusRequestParamsList } from "./params";
+import { CDC_ROUTES } from "./routes";
+
+const Stack = createStackNavigator<CdcBonusRequestParamsList>();
+
+export const CdcStackNavigator = () => (
+  <Stack.Navigator
+    initialRouteName={"CDC_INFORMATION_TOS"}
```

Comment:
```suggestion
    initialRouteName={CDC_ROUTES.INFORMATION_TOS}
```
Since it has the same value, could the usage of a const an improvement?

Reply:
Sure, thank you. Updated here: [9df5c5f](https://github.com/pagopa/io-app/pull/3911/commits/9df5c5fec0d1ae7102d070302a17bba6e825c08b)